modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -274,15 +294,19 @@ let
           lib.optionalString (cfg.config.bindkeysToCode) "--to-code";
       }}
       ${keycodebindingsStr keycodebindings}
-      ${concatStringsSep "\n" (mapAttrsToList inputStr input)}
-      ${concatStringsSep "\n" (mapAttrsToList outputStr output)}
-      ${concatStringsSep "\n" (mapAttrsToList modeStr modes)}
-      ${concatStringsSep "\n" (mapAttrsToList assignStr assigns)}
-      ${concatStringsSep "\n" (map barStr bars)}
-      ${optionalString (gaps != null) gapsStr}
-      ${concatStringsSep "\n" (map floatingCriteriaStr floating.criteria)}
-      ${concatStringsSep "\n" (map windowCommandsStr window.commands)}
-      ${concatStringsSep "\n" (map startupEntryStr startup)}
+      ${concatStringsSep "\n" (
+        # Append all of the lists together to avoid unnecessary whitespace.
+        mapAttrsToList inputStr input # inputs
+        ++ mapAttrsToList outputStr output # outputs
+        ++ mapAttrsToList seatStr seat # seats
+        ++ mapAttrsToList modeStr modes # modes
+        ++ mapAttrsToList assignStr assigns # assigns
+        ++ map barStr bars # bars
+        ++ optional (gaps != null) gapsStr # gaps
+        ++ map floatingCriteriaStr floating.criteria # floating
+        ++ map windowCommandsStr window.commands # window commands
+        ++ map startupEntryStr startup # startup
+      )}
```

Comment:
I added the comments at the end of the lines so that the `./format` script wouldn't put multiple on the same line.

Reply:
Yep, I think that is fine. Since the comment text doesn't really add anything substantial I think you can get away with even just `#` at the end of the line.

Reply:
The formatter is pretty aggressive and removes empty comments, resulting in multiple `mapAttrsToList` in a single line.

Reply:
Right, fair enough.